src/test/java/core/basesyntax/DiamondTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+
+public class DiamondTest {
+    private static final String ONE_STAR = "* \n";
+
+    private static final String EVEN_NUMBER_OF_STAR =
+                      "   * \n"
+                    + "  * * \n"
+                    + " * * * \n"
+                    + "* * * * \n"
+                    + " * * * \n"
+                    + "  * * \n"
+                    + "   * \n";
+
+    private static final String ODD_NUMBER_OF_STAR =
+                      "    * \n"
+                    + "   * * \n"
+                    + "  * * * \n"
+                    + " * * * * \n"
+                    + "* * * * * \n"
+                    + " * * * * \n"
+                    + "  * * * \n"
+                    + "   * * \n"
+                    + "    * \n";
+
+    @Test
+    public void oneStarScenario() {
+        try {
+            StringStream actualPrintStream = new StringStream(new FileOutputStream("file"));
```

Comment:
new FileOutputStream("file") object is a dummy object which is not really used. Let's update StringStream to have empty parameters constructor and use empty contructor here?